src/main/java/ch/njol/skript/effects/EffEating.java

```diff
@@ -0,0 +1,71 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.AbstractHorse;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Panda;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Force Eating")
+@Description("Make a panda or horse type (horse, camel, donkey, llama, mule) start/stop eating.")
+@Example("""
+	if last spawned panda is eating:
+		make last spawned panda stop eating
+	""")
+@Since("INSERT VERSION")
+@RequiredPlugins("Paper (horse type)")
+public class EffEating extends Effect {
+
+	private static final boolean SUPPORTS_HORSES = Skript.methodExists(AbstractHorse.class, "isEating");
+
+	static {
+		Skript.registerEffect(EffEating.class,
+			"make %livingentities% start eating",
+			"force %livingentities% to start eating",
+			"make %livingentities% stop eating",
+			"force %livingentities% to stop eating");
```

Comment:
```suggestion
			"make %livingentities% (start|stop) eating",
			"force %livingentities% to (start|stop) eating",
```
I think you could get away with an option for start/stop. Same goes for the other applicable syntax.